Content.Server/Sound/Components/EmitSoundOnInteractUsingComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared.Sound.Components;
+
+namespace Content.Server.Sound.Components
+{
+    /// <summary>
+    /// Whenever this item is used upon by a specific entity prototype in the hand of a user, play a sound
+    /// </summary>
+    [RegisterComponent]
+    public sealed partial class EmitSoundOnInteractUsingComponent : BaseEmitSoundComponent
+    {
+        [DataField("UsedItemID")]
```

Comment:
this should be upstreamed when possible

Reply:
does this use the right case for component datafields? camel case or something

Reply:
I left a message for Blue, who made this component, to upstream it if he wants. It's in the works

Reply:
I will be making an upstream PR for this component

Reply:
> I will be making an upstream PR for this component

<3 pog thank you

Reply:
PR's been merged, huzzah